packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -13,7 +13,9 @@ import 'mergeable_material.dart';
 import 'theme.dart';
 
 const double _kPanelHeaderCollapsedHeight = kMinInteractiveDimension;
-const double _kPanelHeaderExpandedHeight = 64.0;
+const EdgeInsets _kPanelExpandedDefaultPadding = EdgeInsets.symmetric(
```

Comment:
I would maybe change this to be: 
```dart
const double _kPanelHeaderExpandedDefaultHeight = 64.0;
const EdgeInsets _kPanelHeaderExpandedDefaultPadding = EdgeInsets.symmetric(
    vertical: _kPanelHeaderExpandedHeight - _kPanelHeaderCollapsedHeight,
);
```

That way, future readers of this code will have more context as to how this value was derived.

Reply:
Also, maybe a more descriptive name and a comment might be helpful. Specify that this is the padding for the header of `ExpansionPanel`, or just name it `_kPanelHeaderExpandedDefaultPadding`.

Reply:
Good point. I will adopt your suggestion.